src/UI/Radio.elm

```diff
@@ -151,6 +158,23 @@ withWidth width (RadioGroup prop opt) =
     RadioGroup prop { opt | width = width }
 
 
+{-| `Radio.withDisposition` determines wheter the radio group is laid horizontally or vertically.
```

Comment:
`Radio.withDisposition` determines whether the radio group's items are arranged horizontally or vertically.